snuba/processor.py

```diff
@@ -105,3 +106,18 @@ def _ensure_valid_date(dt):
     if _collapse_uint32(seconds) is None:
         return None
     return dt
+
+
+def _ensure_valid_ip(ip):
```

Comment:
min adding typehints ? we are trying to add them consistently to the snuba code base for new code.